src/insert_sort.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+import timeit
+
+
+best_case = [x for x in range(0, 1000)]
```

Comment:
If you declare constants in the global scope, follow Python convention by making the variable names `ALL_UPPER_CASE`